src/wasm/kotlin/BUILD

```diff
@@ -1,7 +1,15 @@
 load("//build_defs/kotlin_native:build_defs.bzl", "kt_wasm_library")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_library")
 
-kt_wasm_library(
+
+kt_jvm_library(
     name = "arcs",
+    srcs = glob(["src/arcs/Entity.kt"]),
```

Comment:
I feel like this file doesn't belong inside the `wasm` directory if it is generally useful outside of wasm code. There should probably be a general Kotlin folder somewhere, and the wasm-specific stuff can depend on it. Doesn't have to be in this PR, but a TODO at the least would be good.

Reply:
Ray had the idea of adding a `src/sdk` directory (i.e. this could go under `src/sdk/Kotlin`). Is it time to make the move?

Reply:
Let's defer this and discuss in a doc, since there's now a src_kt dir as well, and the copybara script is getting complicated, so we need to rethink repo layout for non-ts